website/routes.py

```diff
@@ -882,6 +882,24 @@ def make_url_map(app):
             project_views.register.node_registration_embargo_disapprove,
             OsfWebRenderer('error.mako', trust=False)
         ),
+        Rule(
+            [
+                '/project/<pid>/registration/approve/<token>/',
+                '/project/<pid>/node/<nid>/registration/approve/<token>/',
+            ],
+            'get',
+            project_views.register.node_registration_approve,
+            OsfWebRenderer('error.mako', trust=False)
```

Comment:
@sloria @samchrisinger there are a lot of cases where this will throw a HTTPError e.g. using a bad token. Informative error messages are handed up the exception chain and populated using error.mako. It's the same pattern we used in both retractions and embargoes.